glue/bin/run-shell

```diff
@@ -40,7 +40,7 @@ shell-component=dbus-update-activation-environment --systemd --verbose WAYLAND_D
 
 shell-ctrl-alt=a:wofi --show drun --location top_left
 shell-meta=a:wofi --show drun --location top_left
-shell-component=wayland-launch swaybg -i /snap/${SNAP_INSTANCE_NAME}/current/usr/share/backgrounds/warty-final-ubuntu.png
+shell-component=wayland-launch swaybg -i ${SNAP}/usr/share/backgrounds/warty-final-ubuntu.png
```

Comment:
This change works here because we always overwrite `${miriway_config}`, but the same code in the classic miriway snap needs to be the current way as it doesn't overwrite. Why have two versions?